public/scss/_forms.scss

```diff
@@ -84,3 +84,87 @@ form.pure-form.pure-form-stacked {
     }
   }
 }
+
+.fieldset {
+  border: 0;
+  padding: 0;
+
+  &__legend {
+    font-weight: 600;
+    font-size: 19px;
+    font-size: 1.1875rem;
+    border-bottom: 0;
+  }
+}
+
+// add this class because I was fighting pure form too much
+.benefits-form {
+  margin-bottom: $space-xl;
+}
+
+.radios {
+  $radioClass: &;
+
+  &__item {
+    font-size: 19px;
+    font-size: 1.1875rem;
+    line-height: 1.25;
+    display: block;
+    position: relative;
+    min-height: 40px;
+    margin-bottom: 10px;
+    padding-left: 40px;
+  }
+
+  &__input {
+    cursor: pointer;
+    position: absolute;
+    z-index: 1;
+    top: -2px;
+    left: -2px;
+    width: 44px;
+    height: 44px;
+    margin: 0;
+    opacity: 0; 
+    
+    &:checked + #{$radioClass}__label::after {
+      opacity: 1;
+    } 
+  }
+
+  &__label {
+    display: inline-block;
+    margin-bottom: 0;
+    padding: 8px 15px 5px;
+    cursor: pointer;
+    -ms-touch-action: manipulation;
+    touch-action: manipulation;
+
+    &::before {
+      content: "";
+      -webkit-box-sizing: border-box;
+      box-sizing: border-box;
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 40px;
+      height: 40px;
+      border: 2px solid currentColor;
+      border-radius: 50%;
+      background: transparent;
+    }
+
+    &::after {
+      content: "";
+      position: absolute;
+      top: 10px;
+      left: 10px;
+      width: 0;
+      height: 0;
+      border: 10px solid currentColor;
+      border-radius: 50%;
+      opacity: 0;
+      background: currentColor;
+    }
+  }
+}
```

Comment:
Looks fairly clean!

The only thing we're missing is the focus styling. I think [this](https://github.com/cds-snc/cra-alpha/blob/master/src/components/forms/MultipleChoice.js#L104) should work. 

What we're looking for is that the currently focused checkbox has a yellow ring around it.